ast-checks/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "nixpkgs-hammer-ast-checks"
+version = "0.0.0"
+authors = ["Robert T. McGibbon <rmcgibbo@gmail.com>"]
+edition = "2018"
+
+[[bin]]
+name = "missing-patch-comment"
+path = "src/missing-patch-comment.rs"
+
+[dependencies]
+rnix = "0.7.0"
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
+codespan = "0.11"
+
+[dependencies.rowan]
+version = "0.6.2"
+features = [ "serde1" ]
```

Comment:
If you're okay using the "normal" syntax for features in the serde declaration above, why bother making a new block for rowan here?

Just do `rowan = { version  = "0.6.2", features = [ "serde1" ] }`